src/DataViews/FunctionsDataView.cpp

```diff
@@ -248,76 +244,50 @@ void FunctionsDataView::OnContextMenu(const std::string& action, int menu_index,
 void FunctionsDataView::DoFilter() {
   m_FilterTokens = absl::StrSplit(absl::AsciiStrToLower(filter_), ' ');
 
-#ifdef WIN32
-  ParallelFilter();
-#else
-  // TODO: port parallel filtering
-  std::vector<uint64_t> indices;
-  const std::vector<const FunctionInfo*>& functions(functions_);
-  for (size_t i = 0; i < functions.size(); ++i) {
-    const FunctionInfo* function = functions[i];
-    std::string name =
-        absl::AsciiStrToLower(orbit_client_data::function_utils::GetDisplayName(*function)) +
-        orbit_client_data::function_utils::GetLoadedModuleName(*function);
-
-    bool match = true;
-
-    for (std::string& filter_token : m_FilterTokens) {
-      if (name.find(filter_token) == std::string::npos) {
-        match = false;
-        break;
-      }
-    }
+  const size_t number_of_threads = thread_pool_->GetPoolSize();
```

Comment:
The size of the thread pool varies and gets adjusted between a minimum and a maximum passed to the constructor, so using this method to decide how many threads to use for filtering seems wrong. Rather, using the max size of the pool would seem more fitting.

Reply:
Or number of cores. The max_thread_pool_size for the app's thread-pool is much bigger then number of processors, this is on purpose because we do not know what kind of tasks are scheduled and we want them to be executed asap.

Maybe in this case we want to create separate thread_pool  limited by number of logical cores.

Reply:
Done.